Zend/zend_ini_parser.y

```diff
@@ -32,7 +32,7 @@
 #include "win32/syslog.h"
 #endif
 
-int ini_parse(void);
+static int ini_parse(void);
```

Comment:
```c
#define yyparse         ini_parse

int
yyparse (void)
{
```

It doesn't seem like this function is declared with `static`.

Reply:
Ok. According to [this](https://stackoverflow.com/a/15670259/1320374), specifying `static` only on the prototype should be ok.